core/src/test/java/org/pentaho/platform/engine/core/system/TestShutdownOrderListener.java

```diff
@@ -0,0 +1,42 @@
+package org.pentaho.platform.engine.core.system;
```

Comment:
You're right. If the existing test code does it this way then it's probably best to just keep as is then.